recipes/metaclock/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "metaclock" %}
+{% set version = "1.0.0" %}
+
+package:
+  name:  "{{ name }}"
+  version:  "{{ version }}"
+
+source:
+  url: https://github.com/SegataLab/metaclock/archive/{{ version }}.tar.gz
+  sha256: cfdefeeb2ad949cff4f2da0b66f2aea226b7a86be63bcb19b9dc5dd7120a78d4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  entry_points:
+    - metaclock_mac = metaclock.metaclock_mac:main
+    - metaclock_visualizer = metaclock.metaclock_visualizer:main
+    - metaclock_tailor = metaclock.metaclock_tailor:main
+    - metaclock_combiner = metaclock.metaclock_combiner:main
+    - bo6_screen.py = metaclock.utils.bo6_screen:main
+    - filter.py = metaclock.utils.filter:main
```

Comment:
Up to you, but do you want to add these commands to the test section as well?

Reply:
thank you for pointing out. commands in line 21 and 22 are dependencies of commands in line 19 and 20 which have been added in the test section. Thus, commands in the line 21 and 22 will be tested as well when line 19 and 20 are tested.